extensions/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/tracing/DropTargetsSampler.java

```diff
@@ -25,15 +25,44 @@ public SamplingResult shouldSample(Context parentContext, String traceId, String
 
         if (spanKind.equals(SpanKind.SERVER)) {
             String target = attributes.get(SemanticAttributes.HTTP_TARGET);
-            // TODO - radcortez - Match /* endpoints
-            if (target != null && dropTargets.contains(target)) {
+            if (shouldDrop(target)) {
                 return SamplingResult.drop();
             }
         }
 
         return sampler.shouldSample(parentContext, traceId, name, spanKind, attributes, parentLinks);
     }
 
+    /**
+     * Determines whether a path should be dropped
+     * TODO: this can certainly be optimized if we find that it's a hot-path
+     */
+    private boolean shouldDrop(String target) {
```

Comment:
Do you think you can add a unit test to this method with some target patterns?

Reply:
Done